lib/cli_common/cli_common/pulse.py

```diff
@@ -47,6 +47,9 @@
 
     queue = 'queue/{}/{}'.format(user, exchange)
     await channel.queue_declare(queue_name=queue, durable=True)
+    await channel.exchange_declare(exchange_name=exchange,
+                                   type_name='topic',
+                                   durable=True)
```

Comment:
@La0 i think this should now fix the problem we had with non existing exchanges when binding a consumer